app/modules/elasticsearch/tasks.py

```diff
@@ -95,3 +94,89 @@ def load_codex_indexes():
   left join "TAXONOMY" as tax on (mi."TAXONOMY_ID_OID" = tax."ID")
 ;
 """
+
+
+def load_encounters_index():
+    wb_engine = create_wildbook_engine()
+    with wb_engine.connect() as wb_conn:
+        results = wb_conn.execute(text(ENCOUNTERS_INDEX_SQL))
+        for result in results:
+            # Create the document object
+            encounter = Encounter(**result)
+            # Assign the elasticsearch document identify
+            encounter.meta.id = f'encounter_{encounter.id}'
+            # Save document to elasticsearch
+            encounter.save(using=current_app.elasticsearch)
+
+
+ENCOUNTERS_INDEX_SQL = """\
+SELECT
+  en."ID" AS id,
+  NULLIF((en."DECIMALLATITUDE"::float || ',' || en."DECIMALLONGITUDE")::text, ',') AS point,
+  en."LOCATIONID" AS locationid,
+  CASE WHEN en."SEX" = 'unk' THEN 'unknown'
+       ELSE en."SEX"
+  END AS sex,
+  en."GENUS" AS genus,
+  an."SPECIES" AS species,
+  en."LIVINGSTATUS" AS living_status,
+  en."LIFESTAGE" AS lifestage
+FROM
+  "ENCOUNTER" AS en
+  LEFT JOIN "ENCOUNTER_ANNOTATIONS" AS ea ON ea."ID_OID" = en."ID"
+  LEFT JOIN "ANNOTATION" AS an ON ea."ID_EID" = an."ID"
+"""
+
+
+def load_sightings_index():
+    wb_engine = create_wildbook_engine()
+    with wb_engine.connect() as wb_conn:
+        results = wb_conn.execute(text(SIGHTINGS_INDEX_SQL))
+        for result in results:
+            # Create the document object
+            sighting = Sighting(**result)
+            # Assign the elasticsearch document identify
+            sighting.meta.id = f'sighting_{sighting.id}'
+            # Save document to elasticsearch
+            sighting.save(using=current_app.elasticsearch)
+
+
+SIGHTINGS_INDEX_SQL = """\
+SELECT
+  oc."ID" AS id,
+  NULLIF((oc."DECIMALLATITUDE"::float || ',' || oc."DECIMALLONGITUDE")::text, ',') AS point,
+  start_time."DATETIME" AS start_time,
```

Comment:
`startTime` and `endTime` have been replaced by the ComplexDateTime `time` value _in houston_.